code/modules/mob/living/carbon/human/species/machine.dm

```diff
@@ -232,3 +232,14 @@
 								)
 		var/error_message = pick(error_messages)
 		to_chat(H, "<span class='boldwarning'>[error_message]</span>")
+
+/datum/species/machine/after_equip_job(datum/job/J, mob/living/carbon/human/H)
+	var/charger_pref = H.client.prefs.active_character.speciesprefs
+	var/obj/item/organ/internal/cyberimp/arm/power_cord/apc/apc_pw = new
+	var/obj/item/organ/internal/cyberimp/arm/power_cord/cell/cell_pw = new
+	if(charger_pref)
```

Comment:
Won't this always give you an APC powercord? Due to strings always being `TRUE`?